src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/metrics/MeanQualityByCycleSparkIntegrationTest.java

```diff
@@ -21,6 +25,46 @@ public String getTestedClassName() {
         return MeanQualityByCycleSpark.class.getSimpleName();
     }
 
+    @DataProvider(name="filenames")
+    public Object[][] filenames() {
+        return new String[][]{
+                {"first5000a.bam", null},
+                {"first5000a.cram", b37_reference_20_21}
```

Comment:
Naming for our test files is so awful ("first5000a"?) -- can you open a ticket to stick to a clear convention for the naming of all test files?